articles/active-directory/saas-apps/github-provisioning-tutorial.md

```diff
@@ -49,10 +49,10 @@ Before configuring and enabling the provisioning service, you need to decide wha
 
 ## Configuring user provisioning to GitHub
 
-This section guides you through connecting your Azure AD to GitHub's user account provisioning API, and configuring the provisioning service to create, update, and disable assigned user accounts in GitHub based on user and group assignment in Azure AD.
+This section guides you through connecting your Azure AD to GitHub's user account provisioning API, and configuring the provisioning service to invite existing GitHub users, update provisioning information, and remove assigned users from your GitHub organization based on user and group assignment in Azure AD.
```

Comment:
the remove statement is correct. We usually don't use the term update provisioning information. Would prefer to say update user account information. Can you clarify why you're not able to create users? the service should be able to create users, not just invite existing users.